chromium_src/PRESUBMIT.py

```diff
@@ -44,3 +45,27 @@ def CheckOverriddenHeadersDeclareIWYUExport(input_api, output_api):
         output_api.PresubmitError(
             f'#include "src/**/*.h" should end with {expected_suffix}', items)
     ]
+
+
+def CheckOverrides(input_api, output_api):
+    items = []
+    with import_inline.sys_path(
+            input_api.os_path.join(input_api.PresubmitLocalPath(), '..',
+                                   'tools')):
+        # pylint: disable=import-error,import-outside-toplevel
+        from chromium_src import check_chromium_src
+        overrides = [
+            f.AbsoluteLocalPath() for f in input_api.AffectedSourceFiles(None)
+            if f.IsTestableFile()
```

Comment:
I think `AffectedSourceFiles` already includes this check.

Reply:
It looks like if you pass a filter into `AffectedSourceFiles` then it does return only testable files, but with `None` (falling back onto the default FilterSourceFile) it doesn't seem to apply testable check.

Reply:
> It looks like if you pass a filter into `AffectedSourceFiles` then it does return only testable files, but with `None` (falling back onto the default FilterSourceFile) it doesn't seem to apply testable check.

hm. what I see:
[`input_api.AffectedSourceFiles`](https://chromium.googlesource.com/chromium/tools/depot_tools.git/+/refs/heads/main/presubmit_support.py#809) calls [`input_api.AffectedTestableFiles`](https://chromium.googlesource.com/chromium/tools/depot_tools.git/+/refs/heads/main/presubmit_support.py#751) (no matter if `FilterSourceFile` is applied or not). `input_api.AffectedTestableFiles` always includes `x.IsTestableFile` check.

maybe I'm missing something?